src/components/file/file.tsx

```diff
@@ -132,32 +130,14 @@ export class File {
     @Element()
     private element: HTMLLimelFileElement;
 
-    @State()
-    private isDraggingOverDropZone = false;
-
-    private fileInput: HTMLInputElement;
-    private fileInputId = createRandomString();
     private chipSet;
     private mdcTextField;
 
-    constructor() {
-        this.handleFileSelection = this.handleFileSelection.bind(this);
-        this.handleFileChange = this.handleFileChange.bind(this);
-        this.handleChipSetChange = this.handleChipSetChange.bind(this);
-        this.handleFileDrop = this.handleFileDrop.bind(this);
-        this.handleKeyDown = this.handleKeyDown.bind(this);
-        this.handleKeyUp = this.handleKeyUp.bind(this);
-        this.handleChipInteract = this.handleChipInteract.bind(this);
-    }
-
     public connectedCallback() {
         this.initialize();
     }
 
     public componentDidLoad() {
-        this.fileInput = this.element.shadowRoot.getElementById(
-            this.fileInputId,
-        ) as HTMLInputElement;
         this.chipSet = this.element.shadowRoot.querySelector(CHIP_SET_TAG_NAME);
         this.initialize();
```

Comment:
Although it's good that you check the entirety of the component but this is old code that I haven't touched :)

Reply:
You can't just remove the connectedCallback. This will break the component if it's ever detached from the DOM and then re-attached. Like if a consumer wants to move the component from one place in the DOM to another.